runtime/bcutil/ROMClassWriter.cpp

```diff
@@ -900,10 +900,7 @@ class ROMClassWriter::Helper :
 			/* Also, encode the arity beyond 1
 			 * (i.e., number of dimensions of the array - 1)
 			 * in the next 2 bytes in Big Endian (since we are maintaining Sun StackMapTable format).
-			 *
-			 * See: https://jtcjazz.ottawa.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/18860
-			 * for explanation of why arity -1 is encoded instead of arity.
-			 *  */
```

Comment:
Can you add the explanation from the internal item?  Better to have the details here in the code rather than losing the link to them.

Reply:
Sure, updated the explanation from the internal item.
@DanHeidinga please have another look.